code/modules/mob/living/basic/clown/clown_ai.dm

```diff
@@ -0,0 +1,22 @@
+/datum/ai_controller/basic_controller/clown
+	blackboard = list(
+		BB_TARGETTING_DATUM = new /datum/targetting_datum/basic,
+		BB_BASIC_MOB_SPEAK_LINES = null,
+	)
+
+	ai_movement = /datum/ai_movement/basic_avoidance
+	idle_behavior = /datum/idle_behavior/idle_random_walk
+	planning_subtrees = list(
+		/datum/ai_planning_subtree/target_retaliate,
+		/datum/ai_planning_subtree/basic_melee_attack_subtree,
+		/datum/ai_planning_subtree/random_speech/blackboard,
+	)
+
+/datum/ai_controller/basic_controller/clown/murder
+	blackboard = list(
+		BB_TARGETTING_DATUM = new /datum/targetting_datum/basic/syndicate, //we use the syndicate targetting datum to avoid repetition AND allow us to murder people
```

Comment:
In that case rename it to `/datum/targetting_datum/basic/attack_until_dead` and move it somewhere else